docs/user-event/intro.mdx

```diff
@@ -11,36 +11,35 @@ events that would happen if the interaction took place in a browser.
 
 These docs describe `user-event@14`. We recommend updating your projects to this
 version, as it includes important bug fixes and new features. You can find the
-docs for `user-event@13.5.0` [here](../ecosystem-user-event.mdx).
+docs for `user-event@13.5.0` [here](../ecosystem-user-event.mdx), and the changelog for the release [here](https://github.com/testing-library/user-event/releases/tag/v14.0.0).
 
 :::
 
 While most examples with `user-event` are for `React`, the library can be used
 with any framework as long as there is a DOM.
 
-## Difference to `fireEvent`
+## Differences with `fireEvent`
 
 The built-in [`fireEvent`](dom-testing-library/api-events.mdx#fireevent) is a
-utility to easily dispatch events. It dispatches exactly the events you tell it
-to and just those - even if those exact events never had been dispatched in a
-real interaction in a browser.
+utility for easily dispatching events. It dispatches exactly the events you tell it
+to and just those - even if those exact events would not have been dispatched in a
+real browser interaction.
 
-`user-event` on the other hand dispatches the events like they would happen if a
-user interacted with the document. That might lead to the same events you
-previously dispatched per `fireEvent` directly, but it also might catch bugs
+`user-event` on the other hand dispatches events as they would be dispatched by a user interacting with the document in a browser. That might lead to the same events you previously dispatched per `fireEvent` directly, but it also might catch bugs
 that make it impossible for a user to trigger said events. This is
 [why you should use `user-event`](https://ph-fritsche.github.io/blog/post/why-userevent)
 to test interaction with your components.
 
 ## Writing tests with `userEvent`
 
-We recommend to use [`userEvent.setup()`](setup.mdx) when rendering your
-component and inline that rendering and setup in your test or use a setup
+We recommend using [`userEvent.setup()`](setup.mdx) when rendering your
+component and inlining that rendering and setup in your test, or using a setup
```

Comment:
Hey, thanks for the improvements.
For me, "inlining" sounds confusing, and I have to take a look at the examples to know what it means. What do you think of the following? 

```suggestion
We recommend invoking [`userEvent.setup()`](setup.mdx) before the component is rendered.
This can be done in the test itself, or by using a setup
```

Reply:
Sure - "inlining" was just carried over from the previous wording, I found it a bit confusing as well.